src/compat/libc/stdlib/wcstombs.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ * @brief wcstombs function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+size_t wcstombs(char *mbstr,  const wchar_t *wcstr, size_t max) {
```

Comment:
If `wcstr` contains terminating `L'\0'`, then terminating `'\0'`-char should be written to `mbstr`.

Reply:
fix it and add test for this situation